sessions/react-custom-hooks/demo-end/src/utils/fetch.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+
+export function useFetch(url) {
```

Comment:
```suggestion
function useFetch(url) {
```

Reply:
i'd keep it to illustrate that this is now something that the whole app could use... even though we're not doing that here :)